crates/dolos/scripts/download.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
What about Windows?

I know there isn't a binary now, but wouldn’t it be more future-proof to write it in Rust, copying the `build.rs` for `testgen-hs`? :)